R/raincloudPlotsWrapper.R

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (C) 2013-2022 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This is a generated file. Don't change it
+
+raincloudPlots <- function(
+    data = NULL,
+    version = "0.19",
+    boxNudge = 0,
+    boxOpacity = 0.5,
+    boxOutline = "colorPalette",
+    boxOutlineWidth = 1,
+    boxPadding = 0.1,
+    boxWidth = 0.1,
+    colorAnyway = TRUE,
+    colorPalette = "colorblind",
+    covariate = "",
+    covariatePalette = "viridis",
+    customAxisLimits = FALSE,
+    customSides = FALSE,
+    customizationTable = list(list(levels = list("species", "island", "bill_length_mm", "bill_depth_mm", "flipper_length_mm", "body_mass_g", "sex", "year"), name = "data 1", values = list("R", "R", "R", "R", "R", "R", "R", "R")), list(levels = list("species", "island", "bill_length_mm", "bill_depth_mm", "flipper_length_mm", "body_mass_g", "sex", "year"), name = "data 2", values = list(0, 0, 0, 0, 0, 0, 0, 0)), list(levels = list("species", "island", "bill_length_mm", "bill_depth_mm", "flipper_length_mm", "body_mass_g", "sex", "year"), name = "data 3", values = list(0, 0, 0, 0, 0, 0, 0, 0))),
```

Comment:
I don't think this default argument makes sense?

Reply:
yes, this wrapper got created from penguins.jasp
in fact, only the three value arguments are relevant for the module.
i simplified the customizationTable default in the wrapper and the unit tests.